lib/gis/parser_json.c

```diff
@@ -28,7 +28,7 @@ char *check_mapset_in_layer_name(char *, int);
 
   The following commands will create according JSON output:
 
-    r.slope.aspect elevation="elevation+https://storage.googleapis.com/graas-geodata/elev_ned_30m.tif" slope="slope+GTiff" aspect="aspect+GTiff" --json
+    r.slope.aspect elevation="elevation@https://storage.googleapis.com/graas-geodata/elev_ned_30m.tif" slope="slope+GTiff" aspect="aspect+GTiff" --json
```

Comment:
`slope+GTiff` and `aspect+GTiff` are fine?

Reply:
Strangely apparently yes.